invenio_app_ils/cli.py

```diff
@@ -259,6 +259,21 @@ def generate(self):
                         "description": "Anti matter"
                     },
                 ],
+                "identifiers": [
+                    {
+                        "scheme": "URN",
+                        "value": "URN:ISBN:0-395-36341-1"
+                    },
+                    {
+                        "scheme": "ISBN",
+                        "value": "0-395-36341-1",
+                        "material": "EBook"
+                    },
+                    {
+                        "scheme": "DOI",
+                        "value": ""
```

Comment:
in general empty value should not happen and if it does, it should raise a ValidationError
checking the empty identifiers case should rather be as:
sometimes the records are created with `identifiers` field filled like you did here 
OR with 
```
`identifiers` = [] # or identifiers not in the record at all
```

to check how the application behaves with empty data field

Reply:
You're absolutely right, I misunderstood what you meant.

Now that you point it out, it seems that the non-emptiness requirement of `value` is not checked by the server (and the problem is the same for document identifiers).

Reply:
I think in the schema it basically thinks that empty string is allowed as a value, when it should not be